Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 group :test do
   gem "fastimage", "~> 2.1.0"
   gem "minitest", "~> 5.10.3"
+  gem "octokit", "~> 4.0"
```

Comment:
Personally would like to see us not bother with any of the `~>` unless there's known breakages as we already get pinned versions from `Gemfile.lock`. Feel free to leave as-is if desired and I'll make a PR to remove them all.

Reply:
🆒 , I opened https://github.com/github/explore/pull/202 and assigned you